_rules/SC2-4-6-descriptive-labels.md

```diff
@@ -95,13 +109,44 @@ Label is visible, but not included in accessibility tree
 
 #### Passed example 5
 
-Label is included in accessibility tree, but not visible
+Label is included in accessibility tree, but not visible.
 
 ```html
 <p id="label_fname" style="position: absolute; top: -9999px; left: -9999px;">First name:</p>
 <input aria-labelledby="label_fname" type="text" name="fname"/>
 ```
 
+#### Passed example 6
+
+Programatically determinable headings provide a context that differentiates the purpose of the otherwise identically named form fields.
+
+```html
+<h2>Shipping address</h2>
+<label>Name<input id="name" type="text" name="name"/></label>
+<label>Street<input id="street" type="text" name="street"/></label>
+
+<h2>Billing address</h2>
+<label>Name<input id="name" type="text" name="name"/></label>
+<label>Street<input id="street" type="text" name="street"/></label>
+````
+
+#### Passed example 7
```

Comment:
Add Passed test case where labels are duplicated on the page but there is a visible context + a different context included in the accessibility tree, and both contexts are descriptive.